hyperliquid/exchange.py

```diff
@@ -49,25 +49,28 @@ def _post_action(self, action, signature, nonce):
         return self.post("/exchange", payload)
 
     def order(
```

Comment:
I'd prefer if we maintained backwards compatibility so that existing users don't need to migrate their code and the examples can use a simpler API where it makes sense. Can you add the batch functionality as a new method called orders (or batch_orders) instead? order can then use the batch api internally to avoid duplicated code